engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -647,8 +645,8 @@ private void execute() throws YarnException, IOException
     int minVcores = conf.getInt("yarn.scheduler.minimum-allocation-vcores", 0);
     LOG.info("Max mem {}m, Min mem {}m, Max vcores {} and Min vcores {} capabililty of resources in this cluster ", maxMem, minMem, maxVcores, minVcores);
 
-    int maxConsecutiveContainerFailures = conf.getInt("MAX_CONSECUTIVE_CONTAINER_FAILURES", MAX_CONTAINER_FAILURES_PER_NODE);
-    long blacklistRemovalTime = conf.getLong("BLACKLIST_REMOVAL_TIME", BLACKLIST_REMOVAL_TIME);
+    int maxConsecutiveContainerFailures = dag.getValue(LogicalPlan.MAX_CONSECUTIVE_CONTAINER_FAILURES);
```

Comment:
Yes, have tested this in cluster. It can be set through conf with this: 
  <property>
    <name>dt.max-consecutive-node-failures</name>
    <value>10</value>
  </property>

Or by directly setting inside populateDag: conf.setInt("dt.max-consecutive-node-failures", 10);